services/movie_session.py

```diff
@@ -0,0 +1,47 @@
+import datetime
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                movie_id=movie_id,
+                                cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: Optional[str] = None) -> QuerySet:
+    if session_date is not None:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
```

Comment:
I recommend to do like that, it future it will be easier to support your code
```suggestion
    queryset = MovieSession.objects.all()
    if session_date is not None:
        queryset = queryset.filter(show_time__date=session_date)
    return queryset
```